deploy/auth-user.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: metrics-server:system:anonymous
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: metrics-server
+subjects:
+- kind: User
+  name: system:anonymous
```

Comment:
This is partly my fault, Piotr borrowed some of my test setup. I had done this locally to make it easy for me to test. I think we should either determine the correct users allowed to read metrics or describe to the customer how to set up read permissions for their users.